wandb/sdk/internal/internal_api.py

```diff
@@ -934,6 +1023,50 @@ def upsert_run(
 
         return response["upsertBucket"]["bucket"], response["upsertBucket"]["inserted"]
 
+    @normalize_exceptions
+    def get_run_info(self, entity, project, name):
```

Comment:
We're missing the docker info that's in `wandb-metadata.json`.  Ideally we would just expose the digest of the container that the experiment was originally ran in when `WANDB_DOCKER` was set in `runInfo`.  Not a blocker for release but should be easy to add...

Reply:
`wandb launch` should also set `WANDB_DOCKER` so we can support customers that are pushing all execution environments to their own registry.

Reply:
Filing a ticket for the first point. And added the WANDB_DOCKER environment variable, but don't see any indication in the UI. Is there anything in the UI? Looking at the runInfo schema, there isn't anything related to docker there. Looks like docker images are associated with projects rather than runs?